codalab/lib/bundle_cli.py

```diff
@@ -1812,6 +1812,16 @@ def do_run_command(self, args):
                 '-T', '--tags', help='Change tags (must appear after worksheet_spec).', nargs='*'
             ),
             Commands.Argument('-d', '--description', help='New bundle description.'),
+            Commands.Argument(
+                '--freeze',
+                help='Freeze bundle to prevent future metadata modification.',
```

Comment:
I feel like "freeze" is associated with "preventing content modification" (i.e., for worksheets), not really "metadata modification", so I'm a bit wary of overloading the term with this PR. But maybe that's okay -- what do you think @nelson-liu ?

Reply:
that's a good point, i'm happy to change it to something else but I'm not really sure waht the right word is.

Reply:
Yeah, I think we're okay with just using "freeze", so let's just keep it.